tests/commands.spec.tsx

```diff
@@ -35,8 +36,10 @@ describe('git-commands', () => {
       commands,
       shell: null as any
     };
+    let translator: ITranslator;
+    const trans = (translator || nullTranslator).load('jupyterlab-git');
```

Comment:
For testing I would recommend using the null translator:

```suggestion
    const trans = nullTranslator.load('jupyterlab-git');
```